### DISCUSSION TITLE
Define a custom Reply-to address for notification emails

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/PublishPress-Planner#1269] [Feature Requests]
Customer ...

> This is a feature request for the Notification Workflows feature (I'm not sure if that's considered part of the Reminders add-on, but I couldn't find an option for this). I would love to be able to define a custom Reply-to address for these notification emails. In my case, the website generates the notification, but it would be really convenient if the person receiving it could hit reply and have their message go to their supervisor instead of the website notification address.

@andergmartins ...

> We could have a field in the notification workflow to configure that. And the value can be a raw email address, or even placeholders for the admin, author, etc

**Comment by andergmartins** [comment]
Note to self: We can try some different things:

https://www.satollo.net/change-wp_mail-behavior-using-hooks

```
add_action('phpmailer_init', 'my_phpmailer_init');
 
function my_phpmailer_init($mailer) {
$mailer->addReplyTo('stefano@mydomain.tld');
}
```

or using wp_mail_from:

https://premium.wpmudev.org/forums/topic/how-to-change-the-replytofrom-email
